### ISSUE TITLE
a11y: Sort options does not have a label, fails accessibility check

### ISSUE THREAD

**Issue by PeterCiuffetti** [appbaseio/reactivesearch#1554]
**Affected Projects**
React

**Library Version:** 3.12.7

**Describe the bug**
The select element produced by the sortOptions of ReactiveList does not pass Axe accessibility check.  It does not have a label.

Axe also complains that the element has insufficient color contrast.  The text in question is dark grey on white, font-weight normal, 1.25em.  This may be a bug in Axe, but Im including it here for reference.

**To Reproduce**
Install the Chrome browser plugin from Deque called Axe
https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd?hl=en

On any page that implements ReactiveList + sortOptions, visit with Chrome.  Open the debugging console and move to the Axe tab.  Press 'Analyze'.

**Expected behavior**
The select element should include an aria-label.  Suggestion:
```
<select class="sort-options css-p4tgvk-sortOptions" name="sort-options" aria-label="Sort options">
<option value="0">Sort by Relevance</option>
<option value="1">Sort by Title</option>
<option value="2">Sort by Date</option>
</select>
```

**Screenshots**
![image](https://user-images.githubusercontent.com/1168204/97218961-be026900-179f-11eb-90ac-a528812bef1f.png)

![image](https://user-images.githubusercontent.com/1168204/97223014-8f878c80-17a5-11eb-9529-ababa5f8f418.png)

**Desktop (please complete the following information):**
 - OS: Mac OS
 - Browser chrome
 - Version 86.0.4240.111
